<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardPageStart</class>
 <widget class="QWizardPage" name="WizardPageStart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Create New</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="radioNewProject">
     <property name="text">
      <string>Project</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioNewCutscene">
     <property name="text">
      <string>Cutscene</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioNewRoom">
     <property name="text">
      <string>Room</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioNewObject">
     <property name="text">
      <string>Object</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radioNewScript">
     <property name="text">
      <string>Script</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
